app/src/docker/Dockerfile

```
@@ -1,7 +1,9 @@
 FROM eclipse-temurin:17-jre-alpine
 
 # hadolint ignore=DL3018
-RUN apk update && apk --no-cache add curl && rm -rf /var/cache/apk/*
+RUN apk update && \
+    apk --no-cache add curl redis sudo && rm -rf /var/cache/apk/* && \
+    apk upgrade openssl libssl3 libcrypto3
```

Delete files after everything. Repeat for other Dockerfiles.
```suggestion
RUN apk update && \
    apk --no-cache add curl redis sudo && \
    apk upgrade openssl libssl3 libcrypto3 && \
    rm -rf /var/cache/apk/*
```

---

Reminder to "Repeat for other Dockerfiles" ☝️